packages/scaffold-config/package.json

```diff
@@ -18,6 +18,7 @@
     "debug": "^4.3.4",
     "fs-extra": "^9.1.0",
     "globby": "^11.0.1",
+    "resolve-package-path": "4.0.3",
```

Comment:
Any reason this is pinned to `4.0.3` rather than doing a `^4.0.3`?

Reply:
No reason, added it manually and forgot to add `^`